### PR TITLE
information about switching settings page location

### PR DESCRIPTION
**Pull request by k1sul1**
I wasted a full 15 minutes figuring out how to move the settings page to a better location, so I added information on how to do that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ The following types require an `options` property. If it is an associative array
 
 A toggle/boolean type field will also use its `options` property when present, but it will default to Off/On. In case of boolean values, the values will require it's arguments in off state/on state order. Keys are ignored.
 
+
```
Comment (nikcorg) [inline README.md]: Remove this redundant whitespace, please.

File: README.md [hunk 2]
```diff
@@ -152,6 +153,33 @@ function validateLength($str, $attribs, $errorCb) {
 }
 ```
 
+## Change settings page location in the menu
+
+By default, `add_plugins_page()` is used for registering the settings page. This places your settings page under the Plugins menu which you may or may not want. Should you want to change the location of the settings page, switch to `add_submenu_page()`.
```
Comment (nikcorg) [inline README.md]: Can you make `add_plugins_page` and `add_submenu_page` links to their pages in the WP documentation.

File: README.md [hunk 3]
```diff
@@ -152,6 +153,33 @@ function validateLength($str, $attribs, $errorCb) {
...
+  'edit.php?post_type=YOUR_CUSTOM_POST_TYPE', // the parent slug which your settings page will use
+  $settings[Settings\S_PAGE_TITLE],
+  $settings[Settings\S_MENU_TITLE],
+  $settings[Settings\S_REQUIRE_CAPS],
+  $settings[Settings\S_PAGE_NAME],
+  getPageRenderer()
+);
+```
+
+All available options can be found in [user contributed notes - add_submenu_page()](https://developer.wordpress.org/reference/functions/add_submenu_page/#user-contributed-notes).
```
Comment (nikcorg) [inline README.md]: Should be

> All applicable options for the parent slug

**Review by nikcorg — CHANGES_REQUESTED** [review]

**Comment by nikcorg** [comment]
Thanks for this. Please adress the slight changes and also add a patch level version bump, after refactoring onto current `master`.

**Comment by k1sul1** [comment]
I made the changes requested.

**Review by nikcorg — APPROVED** [review]

**Pull request merged**